### DISCUSSION TITLE
response is Class:0x00000001057cda20

### DISCUSSION THREAD

**Discussion by stephenhighemprise** [HoneyryderChuck/httpx#39] [Q&A]
After making a request, the result is something like `an instance of #<Class:0x00000001057cda20>` and I was wondering why it wasn't some recognizable class such as a String or a Hash or something else.

**Comment by HoneyryderChuck** [comment]
Ho, do read [those doc](https://gitlab.com/os85/httpx/-/wikis/Response-Handling)
